xf-smoke/src/main/resources/test-content/jcr_root/content/.content.xml

```diff
@@ -26,5 +26,5 @@
     <geometrixx/>
```

Comment:
@stefangrimm  Can we remove all these folders? geometrixx, phonegap, social, we-retail etc? They don't make any sense on skyline and I'm not sure why they were there in the first place.